### DISCUSSION TITLE
Bun? No Thanks!

### DISCUSSION THREAD

**Discussion by babakfp** [oven-sh/bun#9802] [General]
Hi 👋

So, I installed Bun, and:

![Screenshot 2024-04-01 233702](https://github.com/oven-sh/bun/assets/44144724/5bb54497-2539-4340-99a9-afec02cc3e05)

(This ^^^ happened when doing `bun i`).

It took minutes and in the end, it errored that it couldn't resolve the "tailwindcss" package. It didn't look like it was failing again and again or anything like that.

Before that, I tried it on another (smaller) project and it worked. I felt it was slow but I wasn't sure because I'm used to PNPM and cached packages.

Now I try to uninstall Bun but it doesn't even work!

https://github.com/oven-sh/bun/assets/44144724/b573f496-a011-4a36-80e7-5598c561963c

---

## Update

I uninstalled Bun by dragging the `C:\Users\<username>\.bun\uninstall.ps1` file into Powershell (v7) and running it (press `Enter`). The PowerShell that wasn't shown in that video was v1 (the default).

**Comment by Jarred-Sumner** [comment]
Sorry about this. 

> it took minutes

There is a bug when deleting a folder during `bun install` and then attempting to use a folder while its being deleted causes bun to wait indefinitely. That is potentially the cause of this bug.
